requirements.in

```diff
@@ -1,26 +1,60 @@
-# normally pulled in by kombu, but version 2.2.0 is broken
-amqp<=2.1.4
-# also handles Celery as a requirement
-selinon[celery]
-sqlalchemy
-psycopg2
-lxml
-beautifulsoup4
-# We install a patched version from dnf as we cannot use requests from PyPI - we need own certificates
-#requests
+amqp
+anymarkup-core
 anymarkup
+attrs
+beautifulsoup4
+billiard
+boto3
+boto
+botocore
+certifi
+chardet
+click
+codegen
+colorama
+configobj
+docutils
+flake8-polyfill
+flake8
+git2json
+gitdb
+gitpython<=3.1.0
+graphviz
+idna
+importlib-metadata
+jmespath
 jsl
+json5
 jsonschema
-unidiff
-requests
-requests-futures
-git2json
-gitpython
-# Amazon AWS SQS
-# Celery transparently uses boto
-boto
-boto3
-semantic_version
-radon==3.0.1
-watchdog
+kombu
+logutils
+lxml
+mando
+mccabe
+psycopg2-binary
+pycodestyle
+pyflakes
+pyrsistent
+python-dateutil
+pytz
+pyyaml
+radon<=3.0.1
+rainbow-logging-handler
 raven
+requests-futures
+requests
+s3transfer
+selinon[celery]==1.0.0
+semantic-version
+six
+smmap
+soupsieve
+sqlalchemy
+toml<=0.9.4
+urllib3
+vine
+werkzeug
+xmltodict
+zipp
+git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git@21ea911
+git+https://github.com/fabric8-analytics/fabric8-analytics-version-comparator.git@8a57ac7
```

Comment:
This is not used in the worker.

Reply:
utils is used in worker.

Reply:
I meant fabric8-analytics-version-comparator.git is not used.